packages/performance/src/resources/trace.ts

```diff
@@ -158,13 +158,13 @@ export class Trace implements PerformanceTrace {
 
   /**
    * Sets a custom metric to a specified value. Will create a new custom metric if one with the
-   * given name does not exist.
+   * given name does not exist. The value will be floored down to an integer.
    * @param counter Name of the custom metric
    * @param num Set custom metric to this value
    */
   putMetric(counter: string, num: number): void {
```

Comment:
Can we update the function description to mention that only respects the integer part in the type definition please?
https://github.com/firebase/firebase-js-sdk/blob/c1cc428139fa0a934ead91aa955903792d605e23/packages/performance-types/index.d.ts#L74

Reply:
Good idea, done.